app/javascript/components/pages/Home.js

```diff
@@ -1,10 +1,45 @@
 import React from "react"
+import { useNavigate } from "react-router-dom"
+
+const Home = ({ new_user_route, sign_in_route, logged_in }) => {
+  const navigate = useNavigate()
+  const handleSubmit = () => {
+    navigate("/apartmentindex")
+  }
 
-const Home = () => {
   return (
-    <>
-      <h3>Home</h3>
-    </>
+    <div>
+      <h1>Welcome!</h1>
+      <p>
+        lorem ipsum dolor sit amet, consectetur adip lorem ipsum dolor sit amet,
+        consectetur adi lorem ipsum dolor sit amet, consectetur adi lorem ipsum
+        dolor sit amet, consectetur adi
+      </p>
+
+      {/* Show when LOGGED IN */}
+      {logged_in && (
+        <div>
+          <button onClick={handleSubmit} >My Listings</button>
```

Comment:
Question: Do you want all the apartments to be shown for both links: `My Listings` and `View Listings`? My Listings should show the listings created by the current user - protected index. View Listings should be all the listings from the unprotected index.

Reply:
Yeah I see what you mean, I figured that would be something implemented in one of the cards down the line since it wasn't very specific in this or previous cards